packages/webapp/pages/_app.tsx

```diff
@@ -182,17 +181,15 @@ export default function App(props: AppProps): ReactElement {
       <QueryClientProvider client={queryClient}>
         <BootDataProvider app="web" getRedirectUri={getRedirectUri}>
           <SubscriptionContextProvider>
-            <SettingsContextProvider>
-              <OnboardingContextProvider>
-                <AnalyticsContextProvider
-                  app="webapp"
-                  version={version}
-                  getPage={getPage}
-                >
-                  <InternalApp {...props} />
-                </AnalyticsContextProvider>
-              </OnboardingContextProvider>
-            </SettingsContextProvider>
+            <OnboardingContextProvider>
```

Comment:
Onboarding should be removed.
This already happened in the main 260 branch, you might have to rebase

Reply:
I think we should rebase the 260 once this is merged to master as we don't want the current progress at 260 to get merged into production already.

Reply:
Yes, rebase this branch from 260 
The onboarding should be completely gone then (only author onboarding is still a thing)

Reply:
No, I mean, this is pointed at `master` branch since this is safe to be merged. If we rebase this from `260` the current progress of `260` will be included and if we merge this, those features will be included unnecessarily.

Or do you mean we should point this to 260 so we can rebase first?

Reply:
Oh sorry didn't notice this was meant for the master already.
Does make sense yes.

- Merge this into master
- Rebase 260 from master
- ☑️